vcbuild.bat

```diff
@@ -1,4 +1,4 @@
-@echo off
+@if not defined DEBUG_HELPER @ECHO OFF
```

Comment:
YAASS! (I have this floating in all my workspaces, and I keep forgetting to upstream it)